lightning/src/ln/interactivetxs.rs

```diff
@@ -300,33 +300,24 @@ impl InteractiveTxSigningSession {
 		&self.unsigned_tx
 	}
 
-	pub fn counterparty_sent_tx_signatures(&self) -> bool {
-		self.counterparty_sent_tx_signatures
-	}
-
 	pub fn holder_sends_tx_signatures_first(&self) -> bool {
 		self.holder_sends_tx_signatures_first
 	}
 
-	pub fn has_received_commitment_signed(&self) -> bool {
-		self.has_received_commitment_signed
-	}
-
 	pub fn holder_tx_signatures(&self) -> &Option<TxSignatures> {
 		&self.holder_tx_signatures
 	}
 
 	pub fn received_commitment_signed(&mut self) -> Option<TxSignatures> {
-		self.has_received_commitment_signed = true;
 		if self.holder_sends_tx_signatures_first {
 			self.holder_tx_signatures.clone()
 		} else {
 			None
 		}
 	}
 
-	pub fn get_tx_signatures(&self) -> Option<TxSignatures> {
-		if self.has_received_commitment_signed {
+	pub fn get_tx_signatures(&self, received_commitment_signed: bool) -> Option<TxSignatures> {
+		if received_commitment_signed {
```

Comment:
I'm not seeing this used anywhere. Shouldn't we check at the call site instead?

Reply:
Going to remove this, we already get holder signatures and check at call site. Think this was old.